modules/ROOT/pages/configure-message-flows.adoc

```diff
@@ -93,13 +66,43 @@ This is the target message type and the Endpoint where you want to route the tra
 
 * xref:endpoint-https-send.adoc[HTTP and HTTPS Send Configuration]
 
-. After you create the source, map, and target sections in the interface, confirm that each shows a green check mark next to it in the flow), and click *Deploy* to deploy the configurations to your Mule Runtime.
+. After you create the source, map, and target sections in the interface, confirm that each shows a green check mark next to it in the flow, and click *Deploy* to deploy the configurations to your Mule Runtime.
 +
-Your message flow configuration is deployed to your Mule runtime engine and you can start receiving messages from your partner. Use the Activity page to track processed messages.
+. Your message flow configuration is deployed to your Mule runtime engine and you can start receiving messages from your partner.
 +
-Once a message flow is successfully deployed, you can track its processed messages from the Activity page.
+. Once a message flow is successfully deployed, you can track its processed messages from the Activity page.
+
+== Test Message Flows
+
+There are a couple of reasons why you should send test messages through the message flow after you deploy:
+
+* Ensure that you are receiving messages and any acknowledgments from your partners.
+
+* Make sure that the DataWeave map is working and your backend systems receive the transformed message.
+
+You can test the flow by deploying it in a sandbox environment, then undeploying it, and finally deploying it to a production environment.
+
+Test your message flow in your environment:
+
+. Initiate a transmission on the receiving endpoint.
+
+. From within Anypoint Partner Manager select *Activity* and select the transmission from the list.
+
+. Scan the Transmission Flow sections for an exclamation point (!) in any of the rectangle boxes next to the displayed flow stages as this indicates there is an issue with the transmission.
+
+. If view payloads are enabled, review the received payload by clicking *View payload* next to the Received transmission section.
+
+. If you had any TA1 and functional acknowledgments (999 or 997) set, review and confirm that they are there.
```

Comment:
Just checking: is this step part of the previous one (a substep)?
I ask because the previous step does not specify WHAT to check for, and thought this might be it???

If not...how does one confirm the presence of the TA1 acks? Or is that obvious to your audience?

Reply:
I will reword. The "if" statements were to indicate that these choices are optional - depending on your set up. Also, they would know what to look for if they want to download and view payloads. The problem is, that the Sys Admin may or may not have set it up that they can view and download the payloads. I think it's kind of a problem with the design. I added some text, but think I could add a bit more. Yes, the user would know if they should be getting TA1 acknowledgments. The would have set them up when they configured their message flow. It's used in EDI and it indicates the status of the processing.